ethereum-consensus/src/state_transition/error.rs

```diff
@@ -180,6 +183,12 @@ pub enum InvalidVoluntaryExit {
     InvalidSignature(BlsSignature),
 }
 
+#[derive(Debug, Error)]
+pub enum InvalidWithdrawals {
+    #[error("expected withdrawals {expected:?} do not match provided withdrawals {provided:?}")]
```

Comment:
going to use the alt debug format, as it will be much easier to parse the error, if there ever is one

```suggestion
    #[error("expected withdrawals {expected:#?} do not match provided withdrawals {provided:#?}")]
```